types/catalog/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
```

Comment:
Please remove the rules from this tslint. You should just be extending dtslint/dt.json